.github/kokoro/run.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+cd github/$KOKORO_DIR
+
+bazel version
+wget https://github.com/bazelbuild/bazel/releases/download/1.1.0/bazel_1.1.0-linux-x86_64.deb
```

Comment:
I've seen exactly the same error when making the conda package for verible - I solved it by using an older bazel version, see: https://github.com/SymbiFlow/conda-packages/pull/56#discussion_r345240521

Reply:
Although I just noticed that 1.1.0 seems to work in Travis in this particular repo.

It didn't work in the conda-packages repo - maybe it's because of different ubuntu versions? xenial (the default one used for conda-packages) vs bionic used here?